.github/config.yml

```diff
@@ -7,7 +7,7 @@
 
 # https://github.com/behaviorbot/welcome
 newIssueWelcomeComment: >
-  Thank your for raising the issue. We will try and get back to you as soon as possible.
+  Thank you for raising the issue. We will try and get back to you as soon as possible.
```

Comment:
gr8! Welcome to ckblib! @farouk-haroun 
If you are interested in health IT, do checkout my (@dermatologist ) other repos.